src/Domain/System/LogGateway.php

```diff
@@ -113,4 +113,29 @@ public function purgeLogs($title, $cutoffDate)
 
         return $this->db()->delete($sql, $data);
     }
+
+    public function setLog($gibbonSchoolYearID, $module, $gibbonPersonID, $title, $array = null, $ip = null)
```

Comment:
This function is adding a new log entry to the database, but using the verb "set" seem to imply a key-value storage. Wouldn't it better named as "appendLog", "addLog" or "log"?

Reply:
I was thinking that, but I kept it as is to keep inline with the current naming convention. Perhaps a change is for the better though.

Reply:
Thanks Ray and Koala. I agree, updating the method name to align with conventions makes sense. I think `addLog` would work here.

Reply:
Sounds good, I've updated the name accordingly